app/services/staff_action_logger.rb

```diff
@@ -583,6 +583,18 @@ def log_post_rejected(rejected_post, opts = {})
     ))
   end
 
+  def log_webhook(webhook, opts = {})
```

Comment:
I think the parameters for the method can be improved here. We're passing in the action as part of `opts` only to override it with `UserHistory.actions[:webhook]`. For post history events, each action is assigned a key like `post_edit` and `delete_post`. Consolidating all the different events under a single action means that the admin page will only show `webhook` making it difficult to filter between each individual event.

![Screenshot from 2019-03-21 14-59-13](https://user-images.githubusercontent.com/4335742/54737241-f4522c80-4be9-11e9-9c29-050dd9545280.png)